test/functional/page_objects/visualize_page.js

```diff
@@ -656,17 +656,16 @@ export function VisualizePageProvider({ getService, getPageObjects, updateBaseli
     }
 
     async toggleAutoMode() {
-      await testSubjects.click('visualizeEditorAutoButton');
-    }
+      // await testSubjects.click('visualizeEditorAutoButton');
 
-    async sizeUpEditor() {
-      await testSubjects.click('visualizeEditorResizer');
-      await browser.pressKeys(browser.keys.ARROW_RIGHT);
+      // this is a temporary solution, should be replaced with initial after fixing the EuiToggleButton
+      // passing the data-test-subj attribute to a checkbox
+      await find.clickByCssSelector('.visEditorSidebar__controls input[type="checkbox"]');
     }
 
-    async clickOptions() {
-      await find.clickByPartialLinkText('Options');
-      await PageObjects.header.waitUntilLoadingHasFinished();
+    async sizeUpEditor() {
+      await testSubjects.click('splitPanelResizer');
+      await browser.pressKeys(browser.keys.ARROW_LEFT);
```

Comment:
this code won't work on Firefox browser.

This one works for both Chrome & Firefox:
```
async sizeUpEditor() {
      const resizerPanel = await testSubjects.find('splitPanelResizer');
      // Drag panel 100 px left
      await browser.dragAndDrop(
        { location: resizerPanel },
        { location: { x: -100, y: 0 } }
      );
}
```

Reply:
Thanks, updated!